sql/core/src/test/scala/org/apache/spark/sql/PlanStabilitySuite.scala

```diff
@@ -333,3 +340,22 @@ class TPCDSModifiedPlanStabilityWithStatsSuite extends PlanStabilitySuite {
     }
   }
 }
+
+abstract class TPCHPlanStabilitySuiteBase extends PlanStabilitySuite {
```

Comment:
why add an abstract class that has only one child?

Reply:
> why add an abstract class that has only one child?

Have [withState]  subclass before but found not have stats data then remove it.  Remove  TPCHPlanStabilitySuiteBase now